core/src/com/unciv/ui/popups/options/AdvancedTab.kt

```diff
@@ -92,7 +94,22 @@ class AdvancedTab(
             optionsPopup.reopenAfterDisplayLayoutChange()
         }
     }
-
+    
+    private fun addmaxAutosavesStored() {
+        add("Maximum autosave turns stored".toLabel()).left().fillX()
```

Comment:
"autosave turns" makes it seem like this is dependent on the game turns, when actually it's dependent on the number of files
A better phrasing would be "Number of autosaves stored" or perhaps "Number of autosave files stored"